formatTest/typeCheckedTests/expected_output/reasonComments.re

```diff
@@ -1,29 +1,39 @@
 3; /* - */
 
-3;
+3; /*-*/
```

Comment:
Is this PR updated with the latest change that includes the semicolon in the location of the top level binding/eval? If so, I would have expected the comment to be preserved to the left of the semi.